### ISSUE TITLE
Screenshot is taken too soon

### ISSUE THREAD

**Issue by izzyhope58** [jlaitio/livesplit-automatic-split-screenshots#1]
When the screenshot is created, it still shows the timer running and does not show the final time. For example, the timer shows 2.79 when the actual final time is 2.82. Maybe add a setting that lets the user set how long to wait before taking the screenshot or something like that. btw love the idea of this Component, I ALWAYS forget to take screenshots of my splits : )

**Comment by deafmute02** [comment]
> 创建屏幕截图时，它仍显示计时器正在运行，而不显示最终时间。例如，计时器显示 2.79，而实际最终时间是 2.82。也许可以添加一个设置，让用户设置在截取屏幕截图之前要等待多长时间或类似的东西。顺便说一句，我喜欢这个组件的想法，我总是忘记截取我的分割的屏幕截图 : )

Yeah, it's not fixed yet. I saw it recently and nobody fixed it. Also want a feature that takes a screenshot of the timer ahead of time. It'd be like running your best run ever and not finishing it, with a few splits to go, then pressing the shortcut to take a screenshot to share
